selfdrive/car/honda/carcontroller.py

```diff
@@ -148,34 +177,30 @@ def update(self, enabled, CS, frame, actuators,
       lkas_active, CS.CP.carFingerprint, idx, CS.CP.openpilotLongitudinalControl))
 
 
-    accel = actuators.gas - actuators.brake
 
     # TODO: pass in LoC.long_control_state and use that to decide starting/stoppping
     stopping = accel < 0 and CS.out.vEgo < 0.3
     starting = accel > 0 and CS.out.vEgo < 0.3
 
     # Prevent rolling backwards
-    accel = -1.0 if stopping else accel
-    if CS.CP.carFingerprint in HONDA_BOSCH:
-      apply_accel = interp(accel, P.BOSCH_ACCEL_LOOKUP_BP, P.BOSCH_ACCEL_LOOKUP_V)
-    else:
-      apply_accel = interp(accel, P.NIDEC_ACCEL_LOOKUP_BP, P.NIDEC_ACCEL_LOOKUP_V)
+    accel = -4.0 if stopping else accel
 
     # wind brake from air resistance decel at high speed
-    wind_brake = interp(CS.out.vEgo, [0.0, 2.3, 35.0], [0.0, 0.0, 0.15])
+    wind_brake = interp(CS.out.vEgo, [0.0, 2.3, 35.0], [0.001, 0.002, 0.15])
```

Comment:
Why did this change?

Reply:
Hmmmm, this was messing up the interpolation a few lines down at 0 speed. I thought this was causing cruise faults, but in retrospect that was likely something else. I'll make a different PR for this.